views/js/runner/plugins/tools/magnifier/magnifierPanel.js

```diff
@@ -96,6 +105,9 @@ define([
      * @param {Number} [config.levelStep] - The level increment applied when using the controls + and -
      * @param {Number} [config.baseSize] - The base size used to assign the width and the height according to the zoom level
      * @param {Number} [config.maxRatio] - The ratio for the maximum size regarding the size of the window
+     * @param {Object} dynamicComponentConfig - to overrides the default
```

Comment:
Should this comment be here?

Also, noticed that var `zoomSize` is not used, is it a forgotten var?